base/util.go

```diff
@@ -668,6 +668,18 @@ func SingleHostCouchbaseURIToHttpURL(couchbaseUri string) (httpUrl string) {
 
 }
 
+// Take this array of strings and return a single semi-colon delimited string
+// ["http://host1", "http://host2"] -> "http://host1;http://host2"
```

Comment:
I think strings.Join(originalStrings, ";") will handle this, unless there's something I'm missing.